docs/console/03-02-console-backend-service.md

```diff
@@ -20,7 +20,8 @@ These are the available Console Backend Service pluggable modules which contain
 - `apigateway` relates to the API Gateway.
 - `authentication` relates to IDP Presets.
 - `application` relates to the Application Connector.
-- `kubeless` relates to Serverless.
+- `serverless` relates to Serverless.
+- `eventing` relates to Knative Eventing.
 - `rafter` relates to Rafter.
 - `servicecatalog` relates to the Service Catalog, including Service Classes, Service Instances, and Service Bindings.
```

Comment:
```suggestion
- `servicecatalog` relates to the Service Catalog, including ServiceClasses, ServiceInstances, and ServiceBindings.
```

Reply:
In other tutorials we have CR written separately and so I prefer to leave the name separated

Reply:
I also hesitated about that as it's true we have inconsistencies in our documentation, but I talked about it with other TWs and we decided to stick to the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms). 
We'll have a follow-up task to unify it in our docs.